rasa/utils/tensorflow/layers.py

```diff
@@ -580,7 +592,40 @@ def __init__(
         name: Optional[Text] = None,
         parallel_iterations: int = 1000,
         same_sampling: bool = False,
+        constrain_similarities: bool = True,
+        relative_confidence: bool = True,
     ) -> None:
+        """Declare instance variables with default values.
+
+        Args:
+            num_neg: Positive integer, the number of incorrect labels;
+                the algorithm will minimize their similarity to the input.
+            loss_type: The type of the loss function, either 'softmax' or 'margin'.
```

Comment:
```suggestion
            loss_type: The type of the loss function, either 'cross_entropy' or 'margin'.
```
same in the class description

Reply:
Removing the args from the class description since we added to the constructor.